src/sentry/api/serializers/rest_framework/doc_integration.py

```diff
@@ -70,3 +72,26 @@ def create(self, validated_data: MutableMapping[str, Any]) -> DocIntegration:
                 ]
             )
         return doc_integration
+
+    def update(
+        self, doc_integration: DocIntegration, validated_data: MutableMapping[str, Any]
+    ) -> DocIntegration:
+        features = validated_data.pop("features")
+        # Delete any unused features
+        unused_features = IntegrationFeature.objects.filter(
```

Comment:
Does this mean that when you update it you can't add a feature? So if you created it with feature 1, and you want it to have features 1 and 2, you need to put 1 and 2 and not just add 2?

Reply:
Yeah, that is how it's built out right now, it all the on the DocIntegration with whatever came with the request, so the request still needs things like 'name', 'author' 'description' all included as well, even if they don't change. Is that something I should look into changing?

Reply:
Nah I was just making sure I was reading it right - it's gonna be so low volume all we need is clear messaging in _admin.